cmd/lotus-pcr/main.go

```diff
@@ -70,16 +72,24 @@ func main() {
 				EnvVars: []string{"LOTUS_PCR_PATH"},
 				Value:   "~/.lotuspcr", // TODO: Consider XDG_DATA_HOME
 			},
+			&cli.StringFlag{
+				Name:    "log-level",
+				EnvVars: []string{"LOTUS_PCR_LOG_LEVEL"},
+				Hidden:  true,
+				Value:   "info",
+			},
```

Comment:
Note: I'm adding this to this tool as I'm mostly concerned about logs from this tool for debugging and setting this instead of `GOLOG_LOG_LEVEL` is less noisy due to other packages which depends on go-log.

Reply:
Side note: I might file this issue later on go-log, but having a format to change just a single subsystem (or many) from the environment variable of go-log might be nice for development  / debugging

Something like `GOLOG_LOG_LEVEL=miner:info;pubsub:error;*:warn` or something. This can also be easily achieved with jq so not a huge issue for development, but being able to configure this as a static set of systems before system start up might be nice. Make this should live in lotus though as a configuration option.